index.js

```diff
@@ -234,14 +236,15 @@ function SpotifyWebHelper(opts) {
 	};
 	this.player.seekTo = seconds => {
 		this.status.playing_position = seconds; // eslint-disable-line camelcase
+		this.player.emit('seek', seconds);
 		return this.player.play(this.status.track.track_resource.uri + '#' + parseTime(seconds));
 	};
 	this.status = null;
 	var seekingInterval = null;
 	var startSeekingInterval = function () {
 		seekingInterval = setInterval(() => {
-			this.status.playing_position += 0.25; // eslint-disable-line camelcase
-		}, 250);
+			this.status.playing_position += SEEK_INTERVAL_LENGTH / 1000; // eslint-disable-line camelcase
```

Comment:
The original intent was for manual seek events.  If we wanted we could add a separate event for it because I still see use in only listening for manual seeks.  But for the most part it would be better just for the users to have their own interval and just read the status for the current position to update.

If you want I could possibly add another event or at least give an example in the readme of creating your own interval.